src/test/java/io/debezium/connector/spanner/BasicSanityCheckIT.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.spanner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import io.debezium.config.Configuration;
+
+public class BasicSanityCheckIT extends AbstractSpannerConnectorIT {
+
+    private static final String tableName = "embedded_sanity_tests_table";
+    private static final String changeStreamName = "embeddedSanityTestChangeStream";
+
+    @BeforeAll
+    static void setup() throws InterruptedException, ExecutionException {
+        databaseConnection.createTable(tableName + "(id int64, name string(100)) primary key(id)");
+        databaseConnection.createChangeStream(changeStreamName, tableName);
+
+        System.out.println("BasicSanityCheckIT is ready...");
+    }
+
+    @AfterAll
+    static void clear() throws InterruptedException {
+        databaseConnection.dropChangeStream(changeStreamName);
+        databaseConnection.dropTable(tableName);
+    }
+
+    @Test
+    public void shouldNotStartConnectorWithoutRequireConfigs() throws InterruptedException {
+        // Config with only instance id provided.
+        Configuration config = Configuration.create()
+                .with("gcp.spanner.instance.id", database.getInstanceId())
+                .build();
+        start(SpannerConnector.class, config, (success, msg, error) -> {
+            assertThat(success).isFalse();
+            assertThat(msg.contains("Connector configuration is not valid"));
+        });
+        assertConnectorNotRunning();
+    }
+
+    @Test
+    public void shouldNotStartConnectorWithoutNonExistentChangeStreams() throws InterruptedException {
+        final Configuration config = Configuration.copy(baseConfig)
+                .with("gcp.spanner.change.stream", "fooBar")
+                .with("name", tableName + "_test")
+                .with("gcp.spanner.start.time",
+                        DateTimeFormatter.ISO_INSTANT.format(Instant.now()))
+                .build();
+        start(SpannerConnector.class, config, (success, msg, error) -> {
+            assertThat(success).isFalse();
```

Comment:
Is there a msg we can assert here?

Reply:
Added concrete message assertion for this one. Thanks for catching!